src/web/components/Links.tsx

```diff
@@ -81,7 +87,7 @@ const seperator = css`
     }
 `;
 
-const seperatorHide = css`
+const seperatorHideStyles = css`
```

Comment:
I keep thinking we should pre or suf fix css `const`s with `Styles`

Reply:
Yeah, I find without a pre/suffix I mix style variables up with other variables so it's nice to be explicit that something is a css class, and not, say, a boolean.

Reply:
I think we could probably lint this and it'd be worthwhile, it'd improve comprehension a lot